.github/workflows/build.yml

```diff
@@ -270,6 +280,44 @@ jobs:
           rm -rf temp
           mv signed temp
 
+      - name: Generate VSA
+        if: env.shouldSign == 'true'
+        env:
+          AWS_KMS_REGION:  ${{ job.build.steps.auth.with.args.aws-region }}
+          AWS_KMS_KEY_ARN: ${{ job.build.steps.sign.env.AWS_KMS_KEY_ARN }}
```

Comment:
we will need docker credentials for this step to push to $BASHBREW_REFERRERS_REPO too